src/functions/certGen.ts

```diff
@@ -41,6 +41,7 @@ const certGen: Handler = async (
 
   event.Records.forEach((record: SQSRecord) => {
     const testResult: any = JSON.parse(record.body);
+    console.log(`parsed test result is ${JSON.stringify(testResult)}`);
```

Comment:
I know in the lower environments logging the whole test result is not an issue, but when this gets to production it could be.

Would it be worth just logging the systemNumber, that is only useful internally and then we can deduce the test result ourselves from that.

Reply:
I think that is a good idea, we don't want to be logging out peoples names and email addresses. *ITHC glare intensifies* 🤣

Reply:
Agree, we need test result id and system number, from there we can find the record in dynamo